README.md

```diff
@@ -147,6 +147,35 @@ Variable: `ISSUER_PID_SD_JWT_VC_DEFERRED`
 Description: Whether PID issueance in *SD JWT VC* format should be *deferred* or *immediate*  
 Default value: `false` (i.e. immediate issuance)
 
+Variable: `ISSUER_PID_SD_JWT_VC_SIGNING_KEY`  
```

Comment:
I don't think that we need to define so many keys (per credential, per format etc).

I single key pair would be enough.

Reply:
Could the same key be used both for sd-jwt and mdoc signing? 
If both require an ECKey, then we can proceed as you suggest.